cmd/wasm-client/go.mod

```diff
@@ -6,19 +6,18 @@ require (
 	github.com/jessevdk/go-flags v1.4.0
 	github.com/lightninglabs/faraday v0.2.11-alpha
 	github.com/lightninglabs/lightning-node-connect v0.2.3-alpha
-	github.com/lightninglabs/lightning-terminal v0.10.1-alpha.0.20230612093826-ee04747e4a34
-	github.com/lightninglabs/loop v0.24.1-beta
+	github.com/lightninglabs/lightning-terminal v0.10.5-alpha
```

Comment:
Should we bump the version of `lightning-node-connect` to the latest tag (`v0.2.4-alpha`) as well? We do replace the dependency further down, but that way we at least keep it recent.

Reply:
Good point! Addressed with the latest push :)!